libs/shared/ui-components/src/SocialCard/types.ts

```diff
@@ -0,0 +1,71 @@
+import { DomainVariant } from '@quansight/shared/types';
+
+export type TSocialCardProps = {
+  /* 
+  A concise title for the related content.
+  Platform specific behaviors for twitter:
+   iOS, Android: Truncated to two lines in timeline and expanded Tweet
```

Comment:
Also, please fix the indentation so that each line starts on the same column, within each comment.